deconz/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 6.24.0
```

Comment:
Maybe we should bump the version to 7.0.0 here? This seems a rather bigger update which warrants a major version bump. The 6.0.0 bump was also done when moving to a new Debian version.